shell/key-bindings.fish

```diff
@@ -160,21 +171,33 @@ function fzf_key_bindings
     if type -q perl
       set -a FZF_DEFAULT_OPTS '--tac'
       set FZF_DEFAULT_COMMAND 'builtin history -z --reverse | command perl -0 -pe \'s/^/$.\t/g; s/\n/\n\t/gm\''
-    else
+    else if type -q seq
```

Comment:
Are there any practical systems without `seq` but with fzf and fish? Just curious what made you think of that case.

Reply:
Originally, I just wanted to replace `seq` with builtins. But after some benchmarks, I was surprised to find out that the new variation was about 40% slower, so I ended up adding it as fallback instead. But I agree that it probably won't have any practical use case. So, I better remove it then, I wasn't sure about adding it anyway.